docs/how-to-deploy-and-call-a-contract/how-to-deploy-and-call-a-contract.md

```diff
@@ -232,6 +232,54 @@ The major differences between here and [local tests](../how-to-test-a-contract.m
 1. the contract needs to be deployed first;
 2. the contract instance is connected to a real provider, which broadcasts transactions to the blockchain.
 
+### next ()
+The **`next`** property within the **`MethodCallOptions`** interface in sCrypt is used to specify the subsequent contract instance(s) produced in the outputs of the method calling transaction in a stateful contract. This property allows for the chaining of stateful contract calls within a single transaction.
+
+**Where is it used ?**
+
+***In Transaction Builders (txBuilder):***
+
+The **`next`** property is primarily utilized within transaction builders, specifically when dealing with stateful contracts.      
+It is an integral part of the transaction building process when chaining multiple stateful contract calls in a single transaction.
```

Comment:
This line doesn't say much. Change to something like "The transaction builder uses the passed instance(s) to construct outputs of the contract call transaction.".